pkg/kubelet/images/image_gc_manager.go

```diff
@@ -78,6 +79,9 @@ type ImageGCPolicy struct {
 
 	// Minimum age at which an image can be garbage collected.
 	MinAge time.Duration
+
+	// Maximum age after which an image can be garbage collected, regardless of disk usage.
```

Comment:
nit: it would be nice to maybe mention that this is an alpha feature. If feature is disabled, this is set to 0.